marcdown_project/marcdown/api_views/user.py

```diff
@@ -53,5 +53,4 @@ def remove_favorite(self, request):
             profile.favorites.remove(note)
             # TODO: return updated user.get_tags() ??
         else:
-            # TODO: error : unauthenticated
-            pass
+            return JsonResponse(status=status.HTTP_401_UNAUTHORIZED, data={"status" : "false", "message" : "Authentication is required"})
```

Comment:
Could've been a bit more DRY but ok

Reply:
I know. If I had to write it just once more I would have made a function or something. Turns out we are just below the critical mass.